.gitignore

```diff
@@ -83,3 +83,6 @@ lint/generated/
 lint/outputs/
 lint/tmp/
 # lint/reports/
+
+#idea files
+.idea/misc.xml
```

Comment:
also add the entire .idea folder

Like `.idea/`

Reply:
Done

Reply:
Done